fvm/src/call_manager/default.rs

```diff
@@ -96,6 +106,7 @@ where
         method: MethodNum,
         params: Option<Block>,
         value: &TokenAmount,
+        gas_premium: &TokenAmount,
```

Comment:
This looks awkward and leaky here. All parameters to send are **necessary** to execute the send; this is merely contextual information. Let's not put it here, but instead extend the `GasTracker` to hold this value and add a getter for it.

Reply:
ok.

Reply:
done.